lib/renderer/web-frame-init.ts

```diff
@@ -12,6 +12,11 @@ export const webFrameInit = () => {
   ipcRendererUtils.handle('ELECTRON_INTERNAL_RENDERER_WEB_FRAME_METHOD', (
     event, method: keyof WebFrameMethod, ...args: any[]
   ) => {
+    // TODO(MarshallOfSound): Remove once the world-safe-execute-javascript deprecation warning is removed
+    if (method.startsWith('executeJavaScript')) {
+      return (webFrame as any)[`_${method}`](...args);
+    }
```

Comment:
does this... bypass the deprecation warning?

Reply:
Yes, because these results are immediately sent over IPC and not touched in any way it didn't make sense to me to log a deprecation warning for `webContents.executeJavaScript` as no change in behavior would be noticed when the flag is toggled